views/js/runner/provider/qti.js

```diff
@@ -139,7 +139,11 @@ define([
                 var context = self.getTestContext();
                 var states = self.getTestData().states;
                 if(context.state <= states.interacting){
-                    self.loadItem(context.itemUri);
+                    if (context.isTimeout) {
+                        self.timeout();
```

Comment:
This leads on infinite loop like workflow : 

`self.timeout() -> alert -> store -> computeNext -> load -> self.timeout() -> etc.`